gradle.properties

```diff
@@ -1,4 +1,4 @@
-apiversion=1.12.2-R0.1-SNAPSHOT
-skriptapiversion=2.2-dev32
-destinationDir=C:\\Users\\matthew\\Desktop\\VixioSk\\plugins
+apiversion=1.12.2-R0.1-SNAPSHOT
+skriptapiversion=2.2-dev32
+destinationDir=D:\\test\\plugins
```

Comment:
Shouldn't you change this to what it was before?

Reply:
oops i didn't even mean to commit this file